simtools/applications/db_development_tools/add_unit_to_parameter_in_db.py

```diff
@@ -35,14 +35,13 @@ def main():
     for site in ["North", "South"]:
         for par_now, unit_now in zip(pars_to_update, units):
             all_versions = db.get_all_versions(
-                db_name=db.DB_CTA_SIMULATION_MODEL,
                 site=site,
                 parameter=par_now,
                 collection_name="sites",
             )
             for version_now in all_versions:
                 db.update_parameter_field(
-                    db_name=db.DB_CTA_SIMULATION_MODEL,
+                    db_name=None,
```

Comment:
I dont understand this None here. We only allow this application to work with the default staging DB?

Reply:
We actually might never need this application again (as it was used to add units to each parameters in the DB) - we mostly keep it as a reference of how we add fields.

But in general, we need to be flexible with the database names, especially now during the development. To get the model parameter items correct, tests with different databases are necessary and flexibility is therefore important.